### ISSUE TITLE
Atlassian Bitbucket Server Code Insights

### ISSUE THREAD

**Issue by regicsolutions** [fortify-ps/FortifyBugTrackerUtility#70]
Does this utility support Atlassian's Bitbucket Server Code Insights Feature? 

https://confluence.atlassian.com/bitbucketserver/code-insights-966660485.html

Was looking to add a fortify code insights report where users can't merge to master unless the fortify report passes and does not have any med or high annotations.

**Comment by rsenden** [comment]
FortifyBugTrackerUtility currently doesn't support BitBucket, however [FortifyVulnerabilityExporter](https://github.com/fortify/FortifyVulnerabilityExporter) is capable of generating BitBucket reports. Apparently I forgot to document the BitBucket capabilities of FortifyVulnerabilityExporter; see [FortifyVulnerabilityExporter issue 38](https://github.com/fortify/FortifyVulnerabilityExporter/issues/38) for more information.

Would this be suitable for your needs?

**Comment by regicsolutions** [comment]
Do you know if the report it generates is a code insight report that would work with Bitbucket Server?

> FortifyBugTrackerUtility currently doesn't support BitBucket, however [FortifyVulnerabilityExporter](https://github.com/fortify/FortifyVulnerabilityExporter) is capable of generating BitBucket reports. Apparently I forgot to document the BitBucket capabilities of FortifyVulnerabilityExporter; see [FortifyVulnerabilityExporter issue 38](https://github.com/fortify/FortifyVulnerabilityExporter/issues/38) for more information.
> 
> Would this be suitable for your needs?

Do you know if the report it generates is a code insight report that would work with Bitbucket Server?

**Comment by rsenden** [comment]
Not sure, you'd need to check on the Bitbucket side to see whether there are any differences in expected input.

Closing this issue as it's no longer related to FortifyBugTrackerUtility; please open a new issue on the FortifyVulnerabilityExporter repository if necessary.
